tasks/deploy_success.yml

```diff
@@ -1,4 +1,8 @@
 ---
+- name: Load vars
+  include_vars:
+    file: "{{ playbook_dir }}/vendor/roles/slack-notify/vars/main.yml"
```

Comment:
Same here:
```diff
  include_vars:
-  file: "{{ playbook_dir }}/vendor/roles/slack-notify/vars/main.yml"
+  file: "../vars/main.yml"
```

Reply:
Same as [this comment](https://github.com/smithfield-studio/trellis-slack-deploy-notifications/pull/9#discussion_r2078680932)